### PR TITLE
Prettier and husky

### PR DESCRIPTION
**Pull request by MaksymNaboka**

### CODE REVIEW

File: app/(tabs)/_layout.tsx [hunk 1]
```diff
@@ -1,34 +1,41 @@
-import { Tabs } from 'expo-router';
-import React from 'react';
+import { Tabs } from "expo-router";
+import React from "react";
```
Comment (denisorda) [inline app/(tabs)/_layout.tsx:2]: pls, put import from react to first line in all files

File: components/Collapsible.tsx [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
-import { ThemedText } from '@/components/ThemedText';
-import { ThemedView } from '@/components/ThemedView';
-import { Colors } from '@/constants/Colors';
-
-export function Collapsible({ children, title }: PropsWithChildren & { title: string }) {
+import { ThemedText } from "@/components/ThemedText";
+import { ThemedView } from "@/components/ThemedView";
+import { Colors } from "@/constants/Colors";
+import React from "react";
+export function Collapsible({
```
Comment (denisorda) [inline components/Collapsible.tsx:9]: pls, add empty line between imports and functional component

**Review by denisorda — COMMENTED** [review]
You should merge all these commits into several or one and avoid not informative commit messages. Commits tree must be easier for read. For merge pull requests pls use rebase instead merge.

File: .husky/pre-commit [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname "$0")/_/husky.sh"
+echo debug debug debug
+exit 1
```
Comment (avkos) [inline .husky/pre-commit:4]: please remove 
```
echo debug debug debug
exit 1
```

File: app/(tabs)/_layout.tsx (hunk 1 above)
Comment (avkos) [reply app/(tabs)/_layout.tsx:1]: with expo lint we do need to import react in each component

**Review by avkos — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -40,6 +43,9 @@
     "@types/jest": "^29.5.12",
     "@types/react": "~18.2.45",
     "@types/react-test-renderer": "^18.0.7",
+    "eslint": "^8.57.0",
+    "eslint-config-expo": "^7.0.0",
+    "husky": "^9.0.11",
```
Comment (avkos) [inline package.json:48]: I received `sh: prettier: command not found`
add prettier to dev dependencies

**Review by avkos — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (MaksymNaboka) [reply package.json:48]: updated

**Review by avkos — APPROVED** [review]

**Pull request merged**